cmd/grpcurl/grpcurl.go

```diff
@@ -313,6 +320,22 @@ func main() {
 		return cc
 	}
 
+	if *expandHeaders {
+		var err error
+		addlHeaders, err = grpcurl.ExpandHeaders(addlHeaders)
+		if err != nil {
+			fail(err, "Failed to expand additional headers, missing environmental variable")
```

Comment:
nit: "environment variable" instead of "environmental variable"

Reply:
Actually, the error returned from `grpcurl.ExpandHeaders` should suffice so that you don't need the "missing environment variable" bit -- it just makes the error that gets printed unnecessarily longer.